app/Dockerfile

```diff
@@ -1,14 +1,9 @@
-FROM ubuntu:16.04
+FROM ubuntu:20.04
```

Comment:
Again, maybe this should be ubuntu 18 since that is what is running in prod?

Reply:
Agreed - let's stay in sync (I'll revert). It's a bit premature to update until we're actually running 20.04.